packages/test.sh

```diff
@@ -33,7 +33,7 @@ PACKAGE_DIRS=(
     app
     webapps
 )
-RUN_TESTS=(1 1 1 1 1 1 1 1 1 1 1 1 0)
+RUN_TESTS=(1 1 1 1 1 1 1 1 1 1 1 1 1)
 RUN_MYPY=(1 1 1 1 1 1 1 1 1 1 1 1 1)
```

Comment:
I think now we can remove these and the `if`s below, we shouldn't have to regress on purpose.
@jmchilton What do you think?